src/modules/components/dao/daoDataListItem/daoDataListItem.tsx

```diff
@@ -0,0 +1,45 @@
+import type React from 'react';
+import { DataList, Heading, Icon, IconType } from '../../../../core';
+import { DaoAvatar, type IDaoAvatarProps } from '../daoAvatar';
+
+export interface IDaoDataListItemProps extends IDaoAvatarProps {
+    description?: string;
+    daoAddressOrEns?: string;
```

Comment:
I'd suggest to split this property in two properties, we'll be doing the same on other components 
```suggestion
    address?: string;
    ens?: string
```